### DISCUSSION TITLE
U8G2_ST7565_NHD_C12864 shows all pixels on after .begin()

### DISCUSSION THREAD

**Discussion by Paulywog32** [olikraus/u8g2#1826] [Q&A] [answered]
Hi after updating the libraries to the latest version my code now displays all pixels "on" after the .begin() function.
After some testing this issue shows up after version 2.32.7 (works) all newer version display an all black screen.
 
Any help on this would be appreciated. Until then I will be limited to version 2.32.7

I am using the  NHD-C12864A1Z-FSW-FBW-HTT display that has a ST7565 controller. 
using class:
U8G2_ST7565_NHD_C12864_F_4W_SW_SPI
platform IO with ESP32-rover.

Thanks
Paul Cote

![image](https://user-images.githubusercontent.com/4658032/161146098-98781eb5-8d8b-4d61-ae43-854154bf7c78.png)

**Comment by olikraus** [comment]
hmmm... seems to be a feature not a bug: 
https://github.com/olikraus/u8g2/blob/ec7c1fa9a4e2476f59e13fb5317cc8e4bfe95857/csrc/u8x8_d_st7565.c#L644
and
#142 (power down sequence for ST7565, UC1701 and others)

In your code: Did you miss to leave power down mode? in fact power on sequence will put your display into "normal" mode (0xa4):
https://github.com/olikraus/u8g2/blob/ec7c1fa9a4e2476f59e13fb5317cc8e4bfe95857/csrc/u8x8_d_st7565.c#L43-L49

With u8g2 you need to use
```
u8g2.setPowerSave(0)
```
to leave power down mode (https://github.com/olikraus/u8g2/wiki/u8g2reference#setpowersave)

Still strange, because setPowerSave is automatically called with u8g2.begin()

**Comment by Paulywog32** [comment, marked as answer]
Found the issue, there was a change to the contrast value (issue 1678) that caused my old value scaling to not work the same way. sorry for the inconvenience, I've readjusted my value and now the newest version works.

Thanks
